### PR TITLE
[regression-test](fuzzy) fuzzy session variable batch_size

### PR DESCRIPTION
**Pull request by jacktengg**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.6 seconds
 load time: 481 seconds
 storage size: 17170956225 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230203031703_clickbench_pr_89845.html

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run compile

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run feut

**Comment by jacktengg** [comment]
run beut

**Comment by jacktengg** [comment]
run p0

File: regression-test/pipeline/p1/conf/fe.conf [hunk 1]
```diff
@@ -69,4 +69,5 @@ priority_networks=172.19.0.0/24
 enable_outfile_to_local = true
 tablet_create_timeout_second=20
 remote_fragment_exec_timeout_ms=60000
+test_type=p1
```
Comment (yiguolei) [inline regression-test/pipeline/p1/conf/fe.conf:72]: fuzzy_test_type

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 2]
```diff
@@ -1910,6 +1910,14 @@ public class Config extends ConfigBase {
     @ConfField(mutable = false, masterOnly = false)
     public static long external_cache_expire_time_minutes_after_access = 24 * 60; // 1 day
 
+    /**
+     * Github workflow test type, for setting some session variables
+     * only for certain test type. E.g. only settting batch_size to small
+     * value for p0.
+     */
+    @ConfField(mutable = true, masterOnly = false)
+    public static String test_type = "p0";
```
Comment (yiguolei) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1919]: default value should be empty, or our release will use p0 default

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run feut

**Comment by jacktengg** [comment]
run beut

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run beut

**Comment by jacktengg** [comment]
run feut

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**